fuse/plugins/detector_physics/s2_photon_propagation.py

```diff
@@ -317,19 +309,8 @@ def setup(self):
             self.photon_area_distribution
         )
 
-        # Move this part into a nice URLConfig protocol?
         # Field dependencies
-        if any(self.enable_field_dependencies.values()):
-            self.drift_velocity_scaling = 1.0
-            # calculating drift velocity scaling to match total drift time
-            # for R = 0 between cathode and gate
-            if "norm_drift_velocity" in self.enable_field_dependencies.keys():
-                if self.enable_field_dependencies["norm_drift_velocity"]:
-                    norm_dvel = self.field_dependencies_map_tmp(
-                        np.array([[0], [-self.tpc_length]]).T, map_name="drift_speed_map"
-                    )[0]
-                    norm_dvel *= 1e-4
-                    self.drift_velocity_scaling = self.drift_velocity_liquid / norm_dvel
```

Comment:
@HenningSE Notice that I deleted this entire part, which I think was an accidental copypaste from the electron transport plugin.